### ISSUE TITLE
Impossible to create transparent ViewportTexture with Sprite3D

### ISSUE THREAD

**Issue by kubecz3k** [godotengine/godot#11507]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Linux, build 210ebb5cdbecd326c1b0edd35cfda94e3a74c6d9, gpu gtx1060

**Issue description:**
I was not able to create transparent Sprite3D viewport texture, for example to use Label in 3d environment. 

**Steps to reproduce:**
1. Create new scene.
2. Add `Viewport` node 
3. Add `Sprite3D` node
4. Inside `Sprite3D` node  'texture' property create 'new viewport texture', link it with `Viewport` node
5. Ensure 'Transparent' flag is set to true in `Sprite 3D`
6. In `Viewport` node ensure 'Transparent Bg' flag is set to true
7. In `Viewport` node ensure size is bigger than 0,0 (lets say 200x100)
7. Add `Label` node to Viewport node, fill in some text
8. Observe That background is opaque:
![viewport sprite3d](https://user-images.githubusercontent.com/6129594/30772666-6416fbcc-a060-11e7-9908-b8f68b6ac1f9.png)

**Link to minimal example project:**

**Comment by kubecz3k** [comment]
As @n-pigeon noticed it's possible to make it work with setting `Viewport->Render Target -> Clear Mode->true` but it's still very buggy inside editor.

**Comment by n-pigeon** [comment]
Somehow I managed to make it work but it was pain. Still not sure why it works now xD I know I had to sometimes check/uncheck HDR for viewport and setup clear_mode to true. Check/Uncheck for Sprite3D region enable helped sometimes too.
I think there might be bugs related to update and redrawing. Workflow is also a bit tricky :/

![selection_014](https://user-images.githubusercontent.com/6152134/30772980-69bd0e7a-a067-11e7-91d5-28dc09a7a3db.png)
